shared/desktop/yarn-helper/log-to-trace.tsx

```diff
@@ -188,6 +193,9 @@ lines.forEach(line => {
         output.unmatched.push(info)
       }
       break
+    case '|':
```

Comment:
Why do we want to capture pipes if we're ignoring them?

Reply:
because they're valid ones to ignore. it's to skip the default case which is a totally unexpected marker